Tests/Clappr_Tests/Classes/Plugin/Container/SimpleContainerPluginTests.swift

```diff
@@ -0,0 +1,48 @@
+import Quick
+import Nimble
+
+@testable import Clappr
+
+class SimpleContainerPluginTests: QuickSpec {
+    override func spec() {
+        describe(".SimpleContainerPlugin") {
+            context("#init") {
+                it("calls bind events") {
+                    let container = ContainerStub()
+                    let simpleContainerPlugin = SimpleContainerStubPlugin(context: container)
+
+                    expect(simpleContainerPlugin.didCallBindEvents).to(beTrue())
+                }
+
+                it("has a non nil container") {
+                    let container = ContainerStub()
+                    let simpleContainerPlugin = SimpleContainerStubPlugin(context: container)
+
+                    expect(simpleContainerPlugin.container).toNot(beNil())
+                }
+
+                it("has a non nil playback") {
+                    let container = ContainerStub()
+                    let simpleContainerPlugin = SimpleContainerStubPlugin(context: container)
+
+                    expect(simpleContainerPlugin.playback).to(equal(container.playback))
+                }
+
+                it("has a non nil foundationPlayback") {
+                    let container = ContainerStub()
+                    let simpleContainerPlugin = SimpleContainerStubPlugin(context: container)
+
+                    expect(simpleContainerPlugin.foundationPlayback).toNot(beNil())
```

Comment:
Wouldn't it be more interesting to test `foundationPlayback` against `container.playback` as `AVFoundationPlayback`?

Something like this:

```suggestion
                    expect(simpleContainerPlugin.foundationPlayback).to(equal(container.playback as? AVFoundationPlayback))
```